src/OpenLoco/src/Objects/ObjectIndex.cpp

```diff
@@ -56,16 +56,19 @@ namespace OpenLoco::ObjectManager
     static loco_global<ObjectSelectionMeta, 0x0112C1C5> _objectSelectionMeta;
     static loco_global<std::array<uint16_t, kMaxObjectTypes>, 0x0112C181> _numObjectsPerType;
 
-    static constexpr uint8_t kCurrentIndexVersion = 4;
+    static constexpr uint8_t kCurrentIndexVersion = 5;
+    static constexpr uint32_t kMaxStringLength = 1024;
 
     struct ObjectFolderState
     {
         uint32_t numObjects = 0;
         uint32_t totalFileSize = 0;
         uint32_t dateHash = 0;
+        std::string basePath = "";
+
         constexpr bool operator==(const ObjectFolderState& rhs) const
```

Comment:
We can just use following:
```suggestion
        constexpr auto operator<=>(const ObjectFolderState& rhs) const = default;
```